### PR TITLE
(Baymerge) Science Department Overhaul, Adds Senior Scientist Role + Quarters and revamps Science Layout.

### PR DESCRIPTION
**Pull request by Laniakeaaaa**
### What does this add?

**Job:**

**Senior Scientist:**
_Research Director time._
Has access to faxes, being grumpy, keycard authorization and all of the science buttons.

**Mapping:**

- Moves Science Dorms to Deck Two, out of the way and allows for more actual science space.
- Adds Senior Researcher's Office and Quarters in the Bridge Upper Levels and in the Science Xenobotany Wing.
- Adds additional space in the expedition room, minorly tweaks size of the preparation room next to it to accommodate.
- Adds a two tiny items for small details to medbay and conference room. (New Bay toys/desk items)

**Screenshots:**

image
image

**Quarters**
image

**New Office:**
image
**Expedition Storage:**
image
**Science Dorms**
image

### CODE REVIEW

**Comment by Laniakeaaaa** [comment]
I'm unsure as to why, but I've tried multiple versions of the Nerva_Job system and I can't get Senior Researcher to appear at all. Everything else has been tested, however.

File: maps/nerva/datums/nerva_jobs.dm [hunk 1]
```diff
@@ -120,6 +120,27 @@
 /datum/job/firstofficer/get_description_blurb()
 	return "You are the Second Officer, third in command, after the First Officer and the Captain. As the Second Officer, it is your job to oversee personnel issues, which includes managing access, delegating crew grievances, and ensuring the proper upkeep and operation of the ship's recreational and mess facilities. Thus, you are the direct supervisor for the janitorial staff, as well as the culinary and hydroponics staff. As Second Officer, it is also your job to organize and lead awaymissions, and in cases where there is no First Officer present, to pilot the ICS Nerva."
 
+/datum/job/seniorscientist
+	minimal_player_age = 2
+	head_position = 1
+	department = "Science"
+	department_flag = COM|SCI
```
Comment (Naxodile) [inline maps/nerva/datums/nerva_jobs.dm:127]: Jobs default to not being joinable.
```suggestion
	department_flag = COM|SCI
	total_positions = 1
	spawn_positions = 1
```

**Comment by Glloyd** [comment]
I'm fine with this for the most part, although I question the necessity/reasoning for having 2 offices for the senior researcher. I like that they have bridge access, but lorewise I don't think they should have keycard authorization access, and just having the one office in the science wing makes more sense. If you think they should have that room in the bridge, maybe pare it down to a small bedroom? My only other issues are mapping related, which i will go more in detail with below.

File: maps/nerva/datums/nerva_jobs.dm [hunk 2]
```diff
@@ -120,6 +120,28 @@
...
+	minimal_player_age = 2
+	department = "Science"
+	total_positions = 1
+	spawn_positions = 1
+	department_flag = COM|SCI
+	selection_color = "#ad6bad"
+	req_admin_notify = 1
+	economic_power = 15
+	title = "Senior Scientist"
+	supervisors = "Nanotrasen Central Command"
```
Comment (Glloyd) [inline maps/nerva/datums/nerva_jobs.dm:133]: keep the deference to the captain from the scientist info here

File: maps/nerva/datums/nerva_jobs.dm [hunk 3]
```diff
@@ -120,6 +120,28 @@
...
+			 					access_maint_tunnels, access_heads, access_medical, access_ai_upload, access_eva, access_bridge, access_morgue, access_hydroponics,
+								access_library, access_research, access_heads_vault, access_RC_announce, access_keycard_auth, access_gateway, access_expedition_shuttle_helm, access_expedition)
+	minimal_access = list(access_tox, access_tox_storage, access_research, access_xenobiology, access_xenoarch, access_expedition, access_network, access_seniornt,
+			 					access_maint_tunnels, access_heads, access_medical, access_ai_upload, access_eva, access_bridge, access_morgue, access_hydroponics,
+								access_library, access_research, access_heads_vault, access_RC_announce, access_keycard_auth, access_gateway, access_expedition_shuttle_helm, access_expedition)
+
+/datum/job/seniorscientist/get_description_blurb()
+	return "You are a well-respected Senior Scientist working for Nanotrasen's interests. It is your job to manage your science team to ensure that Nanotrasen's research advances. You answer directly to Nanotrasen Central Command itself, which you can fax directly."
```
Comment (Glloyd) [inline maps/nerva/datums/nerva_jobs.dm:144]: and here. it's still the captain's ship after all.

**Comment by Glloyd** [comment]
68747470733a2f2f6d64622e616666656374656461726330372e636f2e756b2f46696c65732f31383030303131362f3537393231393333332f302f61667465722e706e67

**Comment by Laniakeaaaa** [comment]
Merged Anomaly PR, I hope this works, but I imagine Travis is going to cry for some dumbshit reason. Let me know if anything else needs fixed, Airlock's been tagged properly and ID access is all set right.

8e8da31492584c0a2429830fd5d88858
63d4293ab4c21b96b03f87689b0877d8
d22b310ac45466577fe4d0718a8eb03c

**Pull request merged**